packages/cascara/src/specs/Table/Table.spec.mdx

```diff
@@ -1,7 +1,155 @@
+## Table Types
+
+- Data table The standard table layout with a table header and rows of data
+- Grid view - more of a gallery view. No table header. Could be considered a list. Need to discuss if we want to make this a separate component or a variation on a table.
+
+## Table Locations
+
+### Admin
+
+- BRC (Grid)
+- Storefront Designer (grid)
+- Bundles (grid)
+- Role Bundles (grid)
+- Onboarding Activities/Schedule
+- Onboarding Activities/Performance
+- Service Department
+- Service Teams
+- Barista FAQS
+- Barista FAQS/Topics
+- Users
+- Integrations/CSV/Import History
+- Integrations/CSV/Import Schedule
+- Integrations/Email/Barista Inboxes
+- Reports
+- Locations
+
+### Barista
+
+- None
+
+### Caffeine
+
+- Entities
+- Training
+- Variables
+- Logs (future)
+- Users (future)
+- Configurations (future)
+- Localization(future)
+
+### Status
+
+- Packages
+
+### Doppio
+
+- Applications
+- Applications/[Application]
+- Intents
+- Intents/[Intent]
+- FAQs
+- FAQS/[FAQ]
+- Entities
+- Archetype Conditions
+- Attributes
+- Hooks
+- Synonyms
+- Synonyms/[Synonym]
+- Variables
+- Variables/[Variable]
+- Training
+- Spell Corrector Dictionary
+- Utilities
+- ESP Service Departments
+- Service Team Classifications
+- Task Classifications
+- Service Departments
+- Service Teams
+- Task Categories
+- Testing Reports
+- Test Phrases
+- Test Sets
+- Tasks
+- Tasks/[Task]
+- Categories
+- Categories/[Category]
+- Outages
+- Outages/[Outage]
+- Integrations
+- Integrations/[Integration]
+- Service Requests
+- Service Request/[Service Request]
+- Model Bindings
+- Widgets
+- Users
+- Users/[User]
+- Locations
+- Assets
+- Job Roles
+- Classifiers
+- Data Sources
+- Data Sources/[Data Source]
+- Annotations
+- Annotations/[Annotation]
+- Smart Rules
+- Smart Rules/[Rule]
+- Flagged Tickets
+- Surveys
+- Configurations
+- Attachments
+- Business Events
+- Localization
+
+## Existing Table Functionality
+
+### CRUD Capabilities
+
+- Ability to add a data row
+- Ability to edit a table rown (inline optional)
+- Ability to delete a row (optional)
+
+### Table Header
+
+- Column headers with labels and optional descriptions
+- Sortable column headers
+
+### Pagination
+
+- Ability to show total number of records available
+- Ability to show visible number of records
+- Ability to show how many pages of records are available
+- Ability to navigate to a specific page of results
+- Ability to navigate to next or previous page of results
+
+### Table Row
+
+- Ability to have cells/columns with specific defined data structures, form fields, action elements (buttons, menus)
+- Ability for results view to be accessible via URL so results can be shared
+
+## Needed/Suggested/Future Table Functionality
+
+- Zebra striping on rows
+- Ability to bulk edit, delete, add records
+- Table header - horizontal VS vertical
+- Ability to customize columns visible
+- Ability to reorganize position/order of columns
+- Ability to have header/columns be sticky
+- Accordion columns
```

Comment:
This could be P2, if we move what we can to MVP.

Reply:
This is another scenario where I suspect this might be a pattern we actually do not need. Not yet seeing the use cases where it would be needed. But good to have these items here anyway so we can evaluate if our public API would allow us to achieve it if needed later on.